src/sentry/api/event_search.py

```diff
@@ -129,6 +132,16 @@ def is_wildcard(self):
 
 class SearchVisitor(NodeVisitor):
 
+    @cached_property
+    def is_filter_translators(self):
+        is_filter_translators = {
+            'assigned': (SearchKey('unassigned'), SearchValue(False)),
+            'unassigned': (SearchKey('unassigned'), SearchValue(True)),
```

Comment:
So far we haven't used `True/False` values in `SearchValue`. I think it makes sense here, open to changing it if anyone disagrees.

Reply:
makes sense to me, what would the alternative be?

Reply:
No idea, I just noticed that so far we've only used strings, wasn't sure if there was a reason for it.